### PR TITLE
rework blackbox tests

### PR DESCRIPTION
**Pull request by benoit-pierre**
Rework blackbox tests to make it easier to add new ones. Also add tests for bug:

- #719:  pass on master, so it can be closed
- #741: xfail

### CODE REVIEW

File: test/test_blackbox.py [hunk 1]
```diff
@@ -49,209 +103,210 @@ def setUp(self):
...
+        r'''
+        "KWEGS": "question",
+        "-S": "{^s}",
+        "TP-PL": "{.}",
+
+        KWEGS/-S/TP-PL
+
+        "questions. "
+        '''
+        print('arg')
```
Comment (morinted) [inline test/test_blackbox.py]: Extra print statement?

File: test/test_blackbox.py [hunk 2]
```diff
@@ -49,209 +103,210 @@ def setUp(self):
...
+        '''
+
+    @unittest.expectedFailure
+    @simple_replay
+    def test_bug741(self):
+        # Uppercase last word also uppercases next word's prefix.
+        r'''
+        "KPA*TS": "{*<}",
+        "TPAO": "foo",
+        "*ER": "{^er}",
```
Comment (morinted) [inline test/test_blackbox.py]: Extra dictionary entry

File: test/test_blackbox.py (hunk 1 above)
Comment (benoit-pierre) [inline test/test_blackbox.py]: Oops...

File: test/test_blackbox.py (hunk 2 above)
Comment (benoit-pierre) [inline test/test_blackbox.py]: Fixed.

**Pull request merged**